forta-uniswap-event-py/src/agent.py

```diff
@@ -0,0 +1,216 @@
+import binascii
+import json
+
+from forta_agent import Finding, FindingType, FindingSeverity
+from web3 import Web3
+import web3
+
+# Uniswap v2: Router 2
+# 0x7a250d5630b4cf539739df2c5dacb4c659f2488d
```

Comment:
I think the comment containing the address can be removed.  The comment explaining what it is (Uniswap v2: Router 2) could probably also be removed if you choose to change the name of the variable to `UNISWAP_V2_ROUTER_ADDR`, or something similar.